### ISSUE TITLE
JotunnModUnity project have compile errors (maybe tutorials need update?)

### ISSUE THREAD

**Issue by ghost** [Valheim-Modding/JotunnModStub#12]
Hello,

I try to open the Unity Project without success. And The AssetBundle refuse compile build (errors in the editor) : 
```
Library\PackageCache\com.unity.collab-proxy@1.15.15\Editor\Collaborate\Components\SearchBar.cs(41,44): error CS1660: Cannot convert lambda expression to type 'float' because it is not a delegate type
Library\PackageCache\com.unity.collab-proxy@1.15.15\Editor\Collaborate\Components\SearchBar.cs(89,32): error CS1061: 'Timer' does not contain a definition for 'Change' and no accessible extension method 'Change' accepting a first argument of type 'Timer' could be found (are you missing a using directive or an assembly reference?)
```

I try with two tutorials : 
https://github.com/Valheim-Modding/JotunnModStub (Developing Assets with Unity section)
https://valheim-modding.github.io/Jotunn/tutorials/asset-creation.html (mod stub section)

What does it wrong?
I don't have the problem when i create an empty project in unity 2020.3.33f1

**Comment by ghost** [comment]
I try with the mod example, same things occurs : https://github.com/Valheim-Modding/JotunnModExample/tree/master/JotunnUnityExample

In the mod example, we should used DLL from the game directory and not the ripped project.

**Comment by MSchmoecker** [comment]
It's the Unity Collaborate package that causes issues with the asset bundle build process for some reason. It can be removed in the package manager as it's not needed for anything.

I will update the two projects by removing the unnecessary package later.

**Comment by ghost** [comment]
It works, thank you !

**Comment by MSchmoecker** [comment]
This is now fixed in [e40af72](https://github.com/Valheim-Modding/JotunnModStub/commit/e40af720530731d02c5718b48be668d57a3bda12), the packages manifest were actually excluded by the outer Nuget ignore.
